externs/browser/html5.js

```diff
@@ -2043,6 +2043,12 @@ DataTransferItem.prototype.getAsString = function(callback) {};
 DataTransferItem.prototype.getAsFile = function() { return null; };
 
 /**
+ * @return {?Entry} The Entry corresponding to this item, or null.
+ * @nosideeffects
```

Comment:
Is there a link to this specific property in the specs? If so please add a @see link.